package.json

```diff
@@ -97,7 +97,7 @@
     "eslint": "^3.2.2",
     "eslint-plugin-jasmine": "^2.1.0",
     "eslint-plugin-react": "^6.7.0",
-    "jsdoc": "^3.4.0",
+    "jsdoc": "git://github.com/jsdoc3/jsdoc#b21427343c7294bbf1f14c718a390f3e955e37cb",
```

Comment:
What does it gives up?

Reply:
It turns out that jsdoc actually does nothing with @template and the commit adding support for it just made it ignore it instead of throwing an error. I was sort of expecting the fact that the type is generic to appear in the docs somewhere.